packages/next/src/views/Root/index.tsx

```diff
@@ -62,6 +62,11 @@ export const RootPage = async ({
 
   const searchParams = await searchParamsPromise
 
+  // Redirect `${adminRoute}/collections` to `${adminRoute}`
+  if (segments.length === 1 && segments[0] === 'collections') {
+    redirect(adminRoute)
```

Comment:
Either check here before redirecting or move this redirect logic to where the fallback is discovered.